### ISSUE TITLE
Completions are black

### ISSUE THREAD

**Issue by ozanerdem** [oakes/SolidOak#21]
The autocomplete popup window is black, and also the autocomplete options are black except the one which is currently selected. This does not match the screenshot in the repository.

Is there any way to configure these colors?

**Comment by oakes** [comment]
Yeah this is a problem with the Mac version. There must be some terminal color setting that is different than Linux.
